experimental/rust/image/Dockerfile-stack

```diff
@@ -1,22 +1,26 @@
-FROM rust:1.37
+FROM rust:1.39
 
-RUN apt-get update && apt-get install -y lldb
+RUN apt-get update && apt-get install -y lldb musl-tools
+RUN rustup target add x86_64-unknown-linux-musl
+RUN cargo install sccache
 
 ENV CARGO_HOME=/usr/local/cargo/deps
+ENV SCCACHE_DIR=/project/sccache
+ENV RUSTC_WRAPPER=sccache
 
-ENV APPSODY_MOUNTS=.:/project/user-app
+ENV APPSODY_MOUNTS=.:/project/user-app;~/.cache/sccache:/project/sccache
 ENV APPSODY_DEPS=$CARGO_HOME
 
 ENV APPSODY_WATCH_DIR="/project/user-app"
 ENV APPSODY_WATCH_REGEX="^(Cargo.toml|.*.rs)$"
 
-ENV APPSODY_RUN="cargo run"
+ENV APPSODY_RUN="cargo run --target x86_64-unknown-linux-musl"
 ENV APPSODY_RUN_ON_CHANGE="cargo run"
```

Comment:
You need `cargo run --target x86_64-unknown-linux-musl` here too

Reply:
uops